values.yaml

```
@@ -461,71 +619,128 @@ client:
+  #
+  # - `name` - Name of the configMap or secret to be mounted. This also controls
+  #   the path that it is mounted to. The volume will be mounted to `/consul/userconfig/<name>`.
+  #
+  # - `load` - If true, then the agent will be
+  #   configured to automatically load HCL/JSON configuration files from this volume
+  #   with `-config-dir`. This defaults to false.
+  #
+  # @type: array<map>
   extraVolumes: []
-    # - type: secret (or "configMap")
-    #   name: my-secret
-    #   load: false # if true, will add to `-config-dir` to load by Consul
 
   # Toleration Settings for Client pods
   # This should be a multi-line string matching the Toleration array
   # in a PodSpec.
   # The example below will allow Client pods to run on every node
   # regardless of taints
+  #
+  # ```yaml
   # tolerations: |
-  #   - operator: "Exists"
+  #   - operator: Exists
+  # ```
   tolerations: ""
 
   # nodeSelector labels for client pod assignment, formatted as a multi-line string.
   # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  #
   # Example:
+  #
+  # ```yaml
   # nodeSelector: |
   #   beta.kubernetes.io/arch: amd64
+  # ```
+  # @type: string
   nodeSelector: null
 
   # Affinity Settings for Client pods, formatted as a multi-line YAML string.
   # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  #
   # Example:
+  #
+  # ```yaml
   # affinity: |
   #   nodeAffinity:
   #     requiredDuringSchedulingIgnoredDuringExecution:
   #       nodeSelectorTerms:
   #       - matchExpressions:
   #         - key: node-role.kubernetes.io/master
   #           operator: DoesNotExist
+  # ```
+  # @type: string
   affinity: {}
 
-  # used to assign priority to client pods
-  # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  # This value references an existing
+  # Kubernetes `priorityClassName` (https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority)
+  # that can be assigned to client pods.
   priorityClassName: ""
 
-  # Extra annotations to attach to the client pods
-  # Example:
-  #   annotations: |
-  #     "annotation-key": "annotation-value"
+  # This value defines additional annotations for
+  # client pods. This should be a formatted as a multi-line string.
+  #
+  # ```yaml
+  # annotations: |
+  #   "sample/annotation1": "foo"
+  #   "sample/annotation2": "bar"
+  # ```
+  #
+  # @type: string
   annotations: null
 
-  # extraEnvVars is a list of extra environment variables to set with the pod. These could be
-  # used to include proxy settings required for cloud auto-join feature,
-  # in case kubernetes cluster is behind egress http proxies. Additionally, it could be used to configure
-  # custom consul parameters.
+  # extraEnvironmentVars is a list of extra environment variables to set within the stateful set.
```

```suggestion
  # A list of extra environment variables to set within the stateful set.
```
Ignore if unnecessary :)